pkg/trie/triedb/triedb.go

```diff
@@ -17,6 +17,11 @@ import (
 
 var ErrIncompleteDB = errors.New("incomplete database")
 
+type Entry struct {
+	key   []byte
```

Comment:
Do these need to be public for scale?

Reply:
Actually it doesn't need to be public, I changed it now! thks